src/web/app/components/session-edit-form/session-edit-form.component.ts

```diff
@@ -106,11 +106,15 @@ export class SessionEditFormComponent implements OnInit {
   @Output()
   closeEditFormEvent: EventEmitter<void> = new EventEmitter<void>();
 
-  constructor(private simpleModalService: SimpleModalService) { }
+  constructor(private simpleModalService: SimpleModalService, public calendar: NgbCalendar) { }
 
   ngOnInit(): void {
   }
 
+  isToday(data: NgbDateStruct): boolean {
+    const today: NgbDate = this.calendar.getToday();
+    return data.day === today.day && data.month === today.month && data.year === today.year;
```

Comment:
https://ng-bootstrap.github.io/#/components/datepicker/api#NgbDate
You have a `NgbDate` and `NgbDateStruct` type here. `NgnDate.equals(NgbDateStruct)` should be valid here according to the documentations.

Reply:
You might not even need this function all together, since the condition `today` is already provided. https://ng-bootstrap.github.io/#/components/datepicker/api#DayTemplateContext

Reply:
Or you could just omit the custom template and simply style their provided `.ngb-dp-today` class https://ng-bootstrap.github.io/#/components/datepicker/overview#today